exchanges/bitfinex/bitfinex_test.go

```diff
@@ -157,18 +157,13 @@ func TestUpdateOrderExecutionLimits(t *testing.T) {
 		},
 	}
 	for assetItem, pairs := range tests {
-		if err := b.UpdateOrderExecutionLimits(t.Context(), assetItem); err != nil {
-			t.Errorf("Error fetching %s pairs for test: %v", assetItem, err)
+		if assert.NoError(t, b.UpdateOrderExecutionLimits(t.Context(), assetItem), "UpdateOrderExecutionLimits should not error") {
 			continue
 		}
```

Comment:
🚧 This is a require in disguise, isn't it?
🐒 Also looks like this is a `t.Run`
```suggestion
		require.NoError(t, b.UpdateOrderExecutionLimits(t.Context(), assetItem))
```

Reply:
I like to see all the errors rather than cancel on the first instance of one